src/controllers/update/update.landing.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const landingUrl = config.UPDATE_LANDING_PAGE_URL;
```

Comment:
Sorry why is that? 
Again following a similar pattern here to the existing landing page controller.

Reply:
Instead of creating a variable, I would suggest to use the config. Something like this will do 
```
   logger.debugRequest(req, `Redirecting to ${config.UPDATE_LANDING_PAGE_URL}`);
   return res.redirect(config.UPDATE_LANDING_PAGE_URL);
```

Reply:
done